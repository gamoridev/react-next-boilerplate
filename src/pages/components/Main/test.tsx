import { screen, render } from '@testing-library/react'

import Main from '.'

describe('testing <Main /> component', () => {
    it('should render heading', () => {
        const { container } = render(<Main />)

        expect(
            screen.getByRole('heading', { name: /react avan√ßado/i })
        ).toBeInTheDocument()

        expect(container).toMatchSnapshot()
    })
})
