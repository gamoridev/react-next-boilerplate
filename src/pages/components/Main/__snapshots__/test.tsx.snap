// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing <Main /> component should render heading 1`] = `
<div>
  <main>
    <h2>
      React Avan√ßado
    </h2>
  </main>
</div>
`
